act/io/sodar.py

```diff
@@ -0,0 +1,196 @@
+"""
+This module contains I/O operations for loading Sodar files.
+
+"""
+
+import datetime as dt
+import re
+
+import fsspec
+import numpy as np
+import pandas as pd
+
+from act.io.noaapsl import filter_list
+
+
+def read_sodar(filepath):
+    """
+    Returns `xarray.Dataset` with stored data and metadata from a user-defined
```

Comment:
Can you specify what type of sodar is used here? With a link to the instrument docs?

Reply:
Done